filament/backend/include/backend/Handle.h

```diff
@@ -75,6 +75,18 @@ class HandleBase {
     HandleBase(HandleBase const& rhs) noexcept = default;
     HandleBase& operator=(HandleBase const& rhs) noexcept = default;
 
+    HandleBase(HandleBase&& rhs) noexcept
+            : object(rhs.object) {
+        rhs.object = nullid;
+    }
+
+    HandleBase& operator=(HandleBase&& rhs) noexcept {
+        auto t = rhs.object;
+        rhs.object = object;
```

Comment:
This looks like a swap semantic. Do we need to assign the old value to rhs?

Reply:
Actually come to think about it, you're right. That's not a good implementation. Will fix.